core/modules/utils/dom/scroller.js

```diff
@@ -50,7 +50,7 @@ Handle an event
 PageScroller.prototype.handleEvent = function(event) {
 	if(event.type === "tm-scroll") {
 		var options = {};
-		if(event.paramObject && event.paramObject.hasOwnProperty("animationDuration")) {
+		if(event.paramObject && $tw.utils.hop(event.paramObject,"animationDuration")) {
```

Comment:
I should have pointed out that `$tw.utils.hop()` always returns false if the object is falsy, so the first condition can be removed here.

Reply:
apologies for not catching this myself, it has been a while since I looked at that utility method.